dashboard/test/controllers/api/v1/user_school_infos_controller_test.rb

```diff
@@ -561,6 +561,7 @@ def assert_first_tenure(user)
 
     # Then, only update the last confirmation date
     @teacher.reload
+    puts "#{@teacher.user_school_infos.map(&:school_info)}"
```

Comment:
@islemaster This is the test that is failing because school_info_params is not saving the re-assigned value.